packages/target-tauri/src-tauri/src/webxdc/webxdc_scheme.rs

```diff
@@ -99,6 +99,14 @@ pub(crate) fn webxdc_protocol<R: tauri::Runtime>(
     // windows/android:   webxdc://webxdc.localhost/<path>
 
     let webview_label = ctx.webview_label().to_owned();
+    if !webview_label.starts_with("webxdc:") {
+        error!(
+            "prevented other window from accessing webxdc:// scheme (webview label: {})",
+            webview_label
+        );
+        return;
+    }
```

Comment:
same as above, this scheme only works inside of a webxdc. so technically this wouldn't be necessary. however we could also have it for extra clarity.